target/linux/ramips/dts/mt7621.dtsi

```diff
@@ -465,7 +465,7 @@
 	};
 
 	crypto: crypto@1e004000 {
-		compatible = "mediatek,mtk-eip93";
+		compatible = "inside-secure,safexcel-eip93i";
```

Comment:
Looking at the MT7621 logs, all options are implemented. Change this line to
```suggestion
		compatible = "inside-secure,safexcel-eip93ies";
```

Implemented options:
~~~
[   15.277179] inside-secure-eip93 1e004000.crypto: EIP93:3:0:1,PE(0xfe007:0x41710041)
~~~
and coresponding definitions:


~~~
#define   EIP93_PE_OPTION_SHA_256		BIT(19)
#define   EIP93_PE_OPTION_SHA_224		BIT(18)
#define   EIP93_PE_OPTION_SHA_1			BIT(17)
#define   EIP93_PE_OPTION_MD5			BIT(16)
#define   EIP93_PE_OPTION_AES_KEY256		BIT(15)
#define   EIP93_PE_OPTION_AES_KEY192		BIT(14)
#define   EIP93_PE_OPTION_AES_KEY128		BIT(13)
#define   EIP93_PE_OPTION_AES			BIT(2)
#define   EIP93_PE_OPTION_ARC4			BIT(1)
#define   EIP93_PE_OPTION_TDES			BIT(0) /* DES and TDES */
~~~